src/scripts/pages/sidebar.js

```diff
@@ -0,0 +1,45 @@
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
We might want to consider either combining this with `categories.js` or renaming both files (`sidebar.js` or `categories.js`) to better depict what the files are doing. They both touch the same part of the UI, which is why I think they could be in the same file, but they do slightly different things.